pylearn2/models/autoencoder.py

```diff
@@ -727,6 +727,21 @@ def __init__(self, autoencoders):
         self.autoencoders = list(autoencoders)
         self.input_space = autoencoders[0].get_input_space()
         self.output_space = autoencoders[-1].get_output_space()
+        self.corruptor = corruptor
+
+    @functools.wraps(Autoencoder.reconstruct)
+    def reconstruct(self, inputs):
+        """
+        .. todo::
+
+            WRITEME
+        """
```

Comment:
You should not add new "WRITEME" docstrings. Especially in that case, since if you use `wraps`, the docstring of the wrapped method will be used.